compression-service/EXML-converter-service/src/main/java/io/javabrains/EXMLconverterservice/resources/XMLConverter.java

```diff
@@ -0,0 +1,66 @@
+package io.javabrains.EXMLconverterservice.resources;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import com.agiledelta.efx.EFXFactory;
+import com.agiledelta.efx.text.Transcoder;
+import com.agiledelta.efx.EFXException;
+
+
+public class XMLConverter {
+
+    public byte [] compressXML(String xml) throws EFXException, IOException {
+
+            byte[] exmlByteResult = null;
+            try{
```

Comment:
the blank line plus the spacing. I understand it doesn't matter for Java because of the semicolon but it looks off.